perception/sub8_perception/CMakeLists.txt

```diff
@@ -133,6 +133,10 @@ install(DIRECTORY include/${PROJECT_NAME}/
 #     "src/sub8_perception/*.cc"
 # )
 
+file(GLOB START_GATE_SOURCES
+      "src/sub8_perception/start_gate.cc"
```

Comment:
*Removes glasses dramatically* I kind of have to agree with @ironmig here, the style guide is clear and there's no good reason to ignore this part of it.